src/SprykerEco/Shared/Payone/PayoneConfig.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Shared\Payone;
+
+use Spryker\Shared\Kernel\AbstractBundleConfig;
+
+class PayoneConfig extends AbstractBundleConfig
+{
+    public const PAYONE_BANCONTACT_AVAILABLE_COUNTRIES = [
+        'BE' => 'Belgium',
```

Comment:
it's strange that belgium is here already. WHy is it here, and no other countries?

Reply:
Business requirements that only for BE

Reply:
🆗

Reply:
@alex-galych it doesn't matter. We build extendable software, so please do NOT stick to a specific class. 
In this case constant must be protected and the access should be given through a public method.